networking/metallb/metallb-frr-k8s.adoc

```diff
@@ -8,14 +8,12 @@ toc::[]
 
 FRRouting (FRR) is a free, open source internet routing protocol suite for Linux and UNIX platforms.
 `FRR-K8s` is a Kubernetes based DaemonSet that exposes a subset of the `FRR` API in a Kubernetes-compliant manner.
-As a cluster administrator, you can use the `FRRConfiguration` custom resource (CR) to configure `MetalLB` to use `FRR-K8s` as the backend.
+As a cluster administrator, you can use the `FRRConfiguration` custom resource (CR) to configure the way `MetalLB` uses `FRR-K8s`.
```

Comment:
this is more "to leverage the FRR instance beyond the features offered by MetalLB"

Reply:
As a cluster administrator, you can use the FRRConfiguration custom resource (CR) to avail of some of the FRR services not provided by MetalLB, for example, receiving routes.